docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Home
+
+Welcome to our runbook. This aims to be a single source for all Student Robotics
+internal documentation, including information for the [Competition Team][competition-team]
+and [Kit Team][kit-team] as well as other activities.
+
+[competition-team]: https://opsmanual.studentrobotics.org/annual-robotics-competition/competition-programme-team
```

Comment:
I think it would be worth moving these links to the bottom of the file where it's standard to put links. Else inline with the content.

Reply:
I actually quite like having them close to their usages, but I don't feel strongly: 02f0f93